index.html

```diff
@@ -19,8 +19,15 @@
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800' rel='stylesheet' type='text/css'>
 
     <!-- Custom styles for this template -->
-    <link href="css/clean-blog.min.css" rel="stylesheet">
-
+    <link href="css/clean-blog.min.css" rel="stylesheet"> 
+    
+    <meta property="og:title" content="BSDTW" />
+    <meta property="og:description" content="Get involved with BSDTW community" />
+    <meta property="og:url" content="https://bsdtw.org" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="https://bsdtw.org/img/BSDTW17-RGB.png" />
+    <meta property="og:image:width" content="2481" />
+    <meta property="og:image:height" content="2448" />
```

Comment:
Does this really need to be that large? I'm not sure the convention of `og:*` stuff

Reply:
I modified the image size, please check if this size is better.